companies/tasks/dart_sync.py

```
@@ -162,6 +164,40 @@ def sync_all_company_info():
+        f"전체 기업 재무제표 동기화 시작: {total}개 기업, 최근 {years}년 ({current_year - years + 1}~{current_year}년)"
+    )
+
+    task_count = 0
+    for company in companies:
+        try:
+            # 현재부터 과거 N년치 재무제표 동기화 작업 등록
+            for year_offset in range(years):
+                target_year = current_year - year_offset
+                sync_financial_statements.delay(company.stock_code, target_year)
+                task_count += 1
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)